core/snippets/snippets/finallyStatement.snippet

```diff
@@ -14,3 +14,11 @@ language: python
 finally:
     $0
 ---
+
+
+language: r
+-
+finally = {
+    cleanup-code
```

Comment:
This looks like pseudo code that should be replaced by `$0`

Reply:
oops!